src/Adapter/Search/SearchProductSearchProvider.php

```diff
@@ -70,10 +73,14 @@ public function runQuery(
             );
             $products = $result['result'];
             $count    = $result['total'];
+
+            Hook::exec('actionSearch', array('expr' => $queryString, 'total' => $count));
```

Comment:
Like : 
``` php
Hook::exec('actionSearch', array(
    'searched_query' => $queryString, 
    'total' => $count,

    // deprecated since 1.7.x
    'expr' => $queryString, 
));
```

??